region/norfair/east/Nutella Refill.json

```diff
@@ -88,17 +61,105 @@
     }
   ],
   "strats": [
+    {
+      "link": [1, 1],
+      "name": "Leave with Runway",
+      "requires": [],
+      "exitCondition": {
+        "leaveWithRunway": {
+          "length": 3,
+          "openEnd": 1
+        }
+      }
+    },
     {
       "link": [1, 1],
       "name": "Crystal Flash",
       "requires": ["h_canCrystalFlash"]
     },
+    {
+      "link": [1, 2],
+      "name": "Come in Shinecharging, Leave Sparking",
+      "entranceCondition": {
+        "comeInShinecharging": {
+          "length": 2, 
+          "openEnd": 1
+        }
+      },
+      "requires": [
+        "canShinechargeMovement"
+      ],
+      "exitCondition": {
+        "leaveShinecharged": {
+          "framesRemaining": 50
+        }
+      }
+    },
+    {
+      "link": [1, 2],
+      "name": "Come in Shinecharged, Leave Shinecharged",
+      "entranceCondition": {
+        "comeInShinecharged": {
+          "framesRequired": 10
+        }
+      },
+      "requires": [
+        {"shinespark": {"frames": 21}}
+      ],
+      "exitCondition": {
+        "leaveWithSpark": {}
+      }
+    },
     {
       "link": [1, 3],
       "name": "Base",
       "notable": false,
       "requires": []
     },
+    {
+      "link": [2, 1],
+      "name": "Come in Shinecharging, Leave Sparking",
+      "entranceCondition": {
+        "comeInShinecharging": {
+          "length": 2, 
+          "openEnd": 1
+        }
+      },
+      "requires": [
+        "canShinechargeMovement"
+      ],
+      "exitCondition": {
+        "leaveShinecharged": {
+          "framesRemaining": 50
+        }
+      }
+    },
+    {
+      "link": [2, 1],
+      "name": "Come in Shinecharged, Leave Shinecharged",
+      "entranceCondition": {
+        "comeInShinecharged": {
+          "framesRequired": 10
+        }
+      },
+      "requires": [
+        {"shinespark": {"frames": 21}}
```

Comment:
You can't spark from the ground, like in the save rooms. Probably best to jump onto the refill station and then spark